src/informer.ts

```diff
@@ -12,9 +12,13 @@ export type ListPromise<T extends KubernetesObject> = () => Promise<{
     body: KubernetesListObject<T>;
 }>;
 
+// These are issued per object
 export const ADD: string = 'add';
 export const UPDATE: string = 'update';
 export const DELETE: string = 'delete';
+
+// These are issued when a watch connects, reconnects or sees an error
```

Comment:
I see we are only testing for one of these conditions, what about reconnect or errors?

Reply:
Added a test for reconnect. There was already a test for errors. I clarified this comment also.